datadog/tests/cassettes/TestAccRestrictionPolicyUpdate.freeze

```diff
@@ -1 +1,5 @@
-2024-04-03T09:51:45.238776-04:00
\ No newline at end of file
+<<<<<<< Updated upstream
+2024-04-03T09:51:45.238776-04:00
+=======
+2024-04-02T17:42:35.134156-04:00
+>>>>>>> Stashed changes
```

Comment:
that ain't right ;-) some other merge conflict noise in other files

Reply:
oops, note to self to clean that up lol